script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt

```diff
@@ -319,7 +321,7 @@ The school building shudders and shakes.
 [sha:227d2577f98a5d10e3997bdb2caa8bc094bfa9f5]{
 -- Page 278, Offset 32437.
 -- 大気に充満した魔力が連鎖起爆し、校舎の背骨にあたる鉄骨をきしませる。
-Its steel frame groans under the stress caused by the magical energy that had filled the air detonating in a chain reaction.
+Its steel frame groans under the stress caused by the magical energy that had filled the air, detonating in a chain reaction.
```

Comment:
I think this comma incorrectly partitions the sentence. The stress on the building is from the magical energy detonating, but with this comma it kinda reads like the steel frame has detonated in a chain reaction